src/azure-cli/azure/cli/command_modules/eventhubs/commands.py

```diff
@@ -59,6 +65,16 @@ def load_command_table(self, _):
         g.command('delete', 'delete_authorization_rule')
         g.generic_update_command('update', getter_name='get_authorization_rule', setter_name='create_or_update_authorization_rule', custom_func_name='cli_autho_update')
 
+# Cluster Region
+    with self.command_group('eventhubs cluster', eh_clusters_util, resource_type=ResourceType.MGMT_EVENTHUB, client_factory=cluster_mgmt_client_factory) as g:
```

Comment:
add min_api for command group if it is only supported in new api version

Reply:
added min_api to cluster command group